scripts/patch_websockets.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Patches uWebsockets Socket.h to fix a crash sending large
+payloads with ssl encoding.
+"""
+from absl import app
+from absl import flags
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_string("file", None, "Sockets.h file to patch.")
+
+# The line of Socket.h that we intend to start overwriting, and the 
+# number of lines to overwite.
+REPLACE_BLOCK_START_LINE = 308
```

Comment:
I'm a bit confused which part of the code are you trying to replace in Sockets.h.
https://github.com/uNetworking/uWebSockets/blob/4d94401b9c98346f9afd838556fdc7dce30561eb/src/Socket.h#L308

Line 308-325 does not look right to be replaced.

Reply:
I think you are actually replacing 309-326. Worth changing the description and adding the url to uWebSocket code for clarification.

Reply:
Done.